noxfile.py

```diff
@@ -28,6 +28,7 @@
 ALL_INTERPRETERS = ("2.7", "3.6", "3.7")
 PY3_INTERPRETERS = ("3.6", "3.7")
 MAJOR_INTERPRETERS = ("2.7", "3.7")
+BLACK_VERSION = "black==19.10b0"
```

Comment:
New version of black released yesterday caused big diffs that were just formatting. We may start using the new formatting, but will do that in a formatting-only PR.